lms/services/oauth2_token.py

```diff
@@ -44,6 +44,27 @@ def access_token_request(self, token_url, redirect_uri, auth, authorization_code
             },
         )
 
+    def refresh_token_request(self, token_url, redirect_uri, auth, refresh_token):
```

Comment:
I should remove the `refresh_token` argument from this: the method can call `OAuth2TokenService.get()` itself (with no arguments) to get the `refresh_token` itself. (It'll have to raise something if the `refresh_token` is `None`).